samenvatting.tex

```diff
@@ -258,7 +258,7 @@
 			Dit is een expliciete vorm van de trapeziumregel. Het doet eerst een voorspelling met voorwaarts Euler, $\overline w_{j+1} = w_j + h f(t_j,w_j)$. Dan volgt een correctiestap $w_{j+1} = w_j + \frac{h}{2} (f(t_j,w_j) + f(t_{j+1},\overline w_{j+1}))$ .
 			
 		\subsection{Versterkingsfactoren van de methoden}
-			toegepast op $y'=\lambda y,\ y(t_0) = y_0$.
+			Toegepast op $y'=\lambda y,\ y(t_0) = y_0$.
```

Comment:
Not a typo but something to be discussed, I usually keep a capital after a heading, just like the convention in emails after the header